crates/turborepo-ui/src/tui/pane.rs

```diff
@@ -130,8 +125,7 @@ impl<W> TerminalPane<W> {
     }
 
     fn selected(&self) -> Option<(&String, &TerminalOutput<W>)> {
-        let task_name = self.displayed.as_deref()?;
-        self.tasks.get_key_value(task_name)
+        self.tasks.get_key_value(self.displayed_task)
     }
```

Comment:
This can get removed if pass in the just the selected `TerminalOutput` when constructing `TerminalPane`